### ISSUE TITLE
Hi ,  I tried creating ethereum Contract wallet on version 0.9.3 but seems it's hanged on solo network. It's been 24 hr and it still saying creating...

### ISSUE THREAD

**Issue by PramodSingh06** [ethereum/mist#3581]
- [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.0.0`
OS & Version: windows/linux/osx
Node version: `geth 0.0.0` 
Number of blocks synchronized: 0
```

**Comment by hkbrain** [comment]
same for me, stuck on Ropsten
even can't change network

**Comment by lijinchao** [comment]
I find that many people encounter it, and certain one says that 0.9.3 would fix it .but the contract still stuck the condition as you  meet with for the 0.9.3 version. when fixed it ?

**Comment by Kolya56** [comment]
me too, same issue, same version! 5 days running so far!

**Comment by Kolya56** [comment]
mine's on a MacBook running High Sierra, btw.

**Comment by evertonfraga** [comment]
Hi, sorry to hear that.

It is true that there's a bug fix that might solve this "Creating..." situation, but only for Light Client uses. 

Regarding the solo network, you can reset the saved network settings:
1. Go to `%APPDATA%\Mist` config directory
2. locate and delete the `network` file that's hanging there. 
3. Restart mist. 

It should revert to the main net. Please let me know if it worked.

**Comment by ryanio** [comment]
We have released a new version (`0.11.0`) that should be more stable - please give it a try. If you have any new problems that you can reproduce, please create a new issue with the details on how we can help. Thanks!
